### ISSUE TITLE
Custom output directory option produces double slashes in sitemap.xml

### ISSUE THREAD

**Issue by Konakona-chan** [bartholomej/svelte-sitemap#12]
Hi, first of all, thanks for your package, it just works out of the box, which is great.

The small issue I have, as described in title, is that `-o` option produces double slashes in sitemap.xml

My package.json:
```
    "postbuild": "svelte-sitemap --domain https://mydomain.com -o output",
```

My svelte.config.js
```
import preprocess from 'svelte-preprocess';
import adapter from '@sveltejs/adapter-static';

/** @type {import('@sveltejs/kit').Config} */
const config = {
	preprocess: preprocess(),

	kit: {
		adapter: adapter({
			pages: "output",
			assets: "output"
		}),
		target: '#svelte',
		trailingSlash: 'always'
	}
};

export default config;
```

Resulting sitemap (simplified):
```
<?xml version="1.0"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
  <!-- This file was automatically generated by https://github.com/bartholomej/svelte-sitemap v2.0.1 -->
  <url>
    <loc>https://mydomain.com</loc>
  </url>
  <url>
    <loc>https://mydomain.com//about</loc>
  </url>
</urlset>
```

(check the //about URL)

Env info:
```
  System:
    OS: Windows 10 10.0.17763

  npmPackages:
    @sveltejs/adapter-static: ^1.0.0-next.18 => 1.0.0-next.18
    @sveltejs/kit: next => 1.0.0-next.164
    svelte: ^3.34.0 => 3.42.4
    svelte-sitemap: ^2.0.1 => 2.0.1
```

**Comment by felipeqq2** [comment]
This isn't happening with v1.x, though.

**Comment by bartholomej** [comment]
@Konakona-chan @felipeqq2 
You're right! Thank you :) 
Just fixed in version `v2.0.2`

Let me know how it works ;)
